lib/reporters/html.js

```diff
@@ -38,7 +37,7 @@ exports = module.exports = HTML;
 
 var statsTemplate =
   '<ul id="mocha-stats">' +
-  '<li class="progress"><canvas width="40" height="40"></canvas></li>' +
+  '<li class="progress-contain"><progress class="progress-element" max="100" value="0"></progress><svg class="progress-ring"><circle class="ring-whole"/><circle class="ring-highlight" stroke-dasharray="0%,100%"/></svg><div class="progress-text"><em>0</em>%</div></li>' +
```

Comment:
> `<em>`

[Bug] [`<em>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em) indicate _emphasis_. I don't think that's the right semantic implication here. I'm not 100% sure about this, so if you did choose it intentionally I'd love to know why 😄.

Either way, unintended change: the text wasn't italic before, and is getting `font-style: italic` by default now. The old progress bar didn't italicize its text, so the new progress bar shouldn't either.

Reply:
I can't remember exactly, but from what I remember, it's to match with the text values in the template.